x-pack/plugins/enterprise_search/README.md

```diff
@@ -0,0 +1,25 @@
+# Enterprise Search
```

Comment:
Would really appreciate any feedback/copy suggestions - wasn't totally sure what to write for this README file :)

Reply:
I think this is great personally.

Reply:
Ditto